apps/dashboard/src/lib/features/org/api/dash-stats.svelte.ts

```diff
@@ -0,0 +1,21 @@
+import { BaseApi, classroomio } from '$lib/utils/services/api';
+import type { GetDashStatsResponse } from '../utils/types';
+
+/**
+ * API class for organization plan operations
+ */
+class DashStatApi extends BaseApi {
+  /**
+   * Creates a new organization plan
+   * @param params Organization plan creation parameters
+   * @returns Created organization plan
+   */
+  async fetchOrgStats(orgId?: string, siteName?: string) {
+    return this.execute<GetDashStatsResponse>({
```

Comment:
This should be `this.execute<typeof classroomio.dash.stats.$post>({`

Which should get the type of the data coming back from the API.